src/consensus/fork_choice.zig

```diff
@@ -313,6 +318,14 @@ pub const ForkChoice = struct {
         return null;
     }
 
+    pub fn bestSlot(
```

Comment:
This is already implemented as https://github.com/Syndica/sig/blob/cbfbdc8f0ceef85928f246e72194f3108d8dc3e7/src/consensus/fork_choice.zig#L305

I remember during the review that added these methods, it was suggested to change the name. But maybe the original name should have been kept to avoid oversight like this...

Reply:
523c46d2167767976942308aff0823ed8575668c